### ISSUE TITLE
snaptube 

### ISSUE THREAD

**Issue by ngarangan** [imdipprokash/snaptube#1]
what the type of this application? is video downloader like snaptube?

**Comment by kittyjhon89-dev** [comment]
Yes, it is basically a [video downloader](https://snaptuebe.com/) application similar to Snaptube. It allows users to download videos from different platforms and watch them offline. It may also include extra features like a built-in browser, media player, or audio download option.
